app/notification.go

```diff
@@ -365,7 +365,7 @@ func (a *App) SendNotifications(post *model.Post, team *model.Team, channel *mod
 
 		var infos []*model.FileInfo
 		if result := <-fchan; result.Err != nil {
-			mlog.Warn(fmt.Sprint("api.post.send_notifications.files.error FIXME: NOT FOUND IN TRANSLATIONS FILE", post.Id, result.Err), mlog.String("post_id", post.Id))
+			mlog.Warn(fmt.Sprint("Faile to get the fileInfo", post.Id, result.Err), mlog.String("post_id", post.Id))
```

Comment:
Is this for files uploaded to Mattermost

Reply:
to add the images info in the push notification